src/main/java/com/amalitech/gpuconfigurator/repository/OtpRepository.java

```diff
@@ -0,0 +1,13 @@
+package com.amalitech.gpuconfigurator.repository;
+
+import com.amalitech.gpuconfigurator.model.Otp;
+import org.springframework.data.repository.CrudRepository;
+
+import java.util.Optional;
+import java.util.UUID;
+
+public interface OtpRepository extends CrudRepository<Otp, UUID> {
```

Comment:
please can we also extend JpaRepository for our Repos instead of CrudRepositories,